dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/DefaultEventService.java

```diff
@@ -228,16 +219,8 @@ private Event getEvent(
     event.setEventDataValues(dataValues);
 
     if (eventParams.isIncludeRelationships()) {
```

Comment:
Can we move this if into the `getEvents(operationParams, new PageParams(1, 1, false))`? I think we can have one `get` method in every service that does interface with stores. The other `get` variants job is to create the `EventOperationParams` and call the one method.

Reply:
Yes, this if is not necessary here at all. We are already mapping the relationships in the `getEvents` method